sensei-lms.php

```diff
@@ -47,7 +47,7 @@
 	define( 'SENSEI_LMS_PLUGIN_PATH', plugin_dir_path( SENSEI_LMS_PLUGIN_FILE ) );
 }
 
-if ( class_exists( 'Sensei_Main' ) ) {
+if ( class_exists( 'Sensei_Main' ) && ! defined( 'PHPUNIT_SENSEI_TESTSUITE' ) ) {
```

Comment:
The `Sensei_Main` class is always loaded in a PHPUnit environment. This is a side-effect of using the Composer autoloader.

Reply:
This works, but also using this seems to fix it. I can't think of a situation where we'd want to prevent autoloading it outside of testing, though.

```suggestion
if ( class_exists( 'Sensei_Main', false ) ) {
```